### PR TITLE
Added a Workflow entry, specifically regarding the team's standup policy.

### PR DESCRIPTION
**Pull request by vulcansmithy**
Added the Dev Teams Standup workflow.

### CODE REVIEW

File: workflow/standup.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Standup
+
+Standup will be conducted everyday starting ``10:30am`` Monday to Friday. The venue for the virtual Standup will be conducted over ``Google Meet``.
```
Comment (Erol) [inline workflow/standup.md]: How about we post the Meet link as well?

Comment (vulcansmithy) [reply workflow/standup.md]: Good idea. Let me add that.

Comment (vulcansmithy) [reply workflow/standup.md]: Recommendation added.

**Review by JonArthur — APPROVED** [review]

File: standup.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Standup
+
+Standup will be conducted everyday starting ``10:30am`` Monday to Friday. The venue for the virtual Standup will be conducted over [Google Meet](https://meet.google.com/sjr-pjar-mjq).
```
Comment (maurusrv) [inline standup.md:3]: Just a comment: the url might change for some reason (like scheduling a new one next month). I've explored it in google calendar, and I think we can personalize the url to be recognizable than an auto-generated one. 

Not sure, but mentioned it anyway as we might need to update this every time if it is indeed the case.

Comment (maurusrv) [reply standup.md:3]: Oops. checked it again. one can only enter an existing meeting code but still i'm not sure. But i guess this won't change.

**Review by maurusrv — APPROVED** [review]

**Pull request merged**